### PR TITLE
[READY] Badmin Gauntlet

### LINKED ISSUE #9145: [SUGGESTION] The Wizard's Badmin Gauntlet (Infinity Gauntlet) + Declaration of Wizardry Warfare!
## Badmin Gauntlet
(it's actually just the Infinity Gauntlet but with some things changed so we don't get sued by disney and we're within grounds of parody)
Also I'll be the one coding this, mainly. At least until I realize I have no idea what I'm doing and ask for help.
thanos

[Implementation]: 
The Badmin Gauntlet will be an "Artifact of Power" sold to the Wizard, similar to the Book of Darkness. It should cost 10 points, i.e all of the Wizards points, and simply spawns a paper. The paper will explain that the Badmin Gauntlet is a _declaration of war_ and spawn a special box. If they hit the Spell Book with the box, they will get all of their points back and they can go back to normal. If they open the special box, an announcement (similar to the Nanosuit) will be made along with a message for a declaration of war from the Wizard's federation. The wizard will be equipped with the Badmin Gauntlet and some base spells that are more meant for general versatility and power _before_ he gets the stones. When he gets the stones, some things change.
The basis of the idea is that the Wizard is able to cast spells and dish out huge damage with the gauntlet simultaneously, creating an incredibly overpowered antagonist truly deserving of a Declaration of War.

### The Gauntlet
The Gauntlet will take up both a glove spot and your right hand, where both are nodrop. The gloves will protect you from shocks and fire. The Gauntlet in your right hand will function as your means of actually dealing damage, swapping modes with the Gauntlet, and so on.
Intents in every mode generally do something different. But, Gauntlet starts off with no extra "modes", it's just your Gauntlet. "Modes" are unlocked by grabbing the extra stones which makes you gradually more powerful.

All of the modes are completely possible with the current code and actually would not require too much fiddling. It's not my fault it'll end up being around a thousand lines long regardless, Carbon.

Mode 1: Default (No Stones) / Syndie Stone
The Default Gauntlet is available only when you don't have the Antag Stone. It boasts Martial Arts combos, huge damage, 50% ranged deflects, and the ability to punch down walls with some startup time. Using help intent on someone who is downed will have you try to break one of their legs, but if they're standing you'll merely shove them.
Once you get the Syndie Stone, Default is made much stronger. Punching down walls is now instant, combos deal much more damage, and you will now stun on punch similar to Hulks. On top of that, using help on someone who is downed will let you rip off their leg and use it as a long ranged weapon, or even throw it to deal more damage.

Mode 2: Bluespace Stone
The Bluespace Stone is mostly helpful for getting around quickly. Most of its use comes from the spells it grants, but you can still use the gauntlet for combat purposes.
Help Intent: Warp Whistle effect. Can be used on yourself, or your enemies!
Harm Intent: Quickly warp where you were clicking. Drains some stamina.
Grab Intent: No changes. Suggest some!
Disarm intent: Knocks their weapons straight into your backpack! If you don't have the room, teleports said item to a random room.

Mode 3: Server(? name W.I.P.) Stone.
Not much, but worthwhile.
Using the mode on walls will turn the walls into completely random objects. If used on things like Cloth, it can turn it into golems. Useful for amassing armies out of the salted ghosts.
All intents but Harm: Fire a bolt of animation.
Harm Intent: Staff of Chaos. As things go with the Staff of Chaos, this will either end amazingly or terribly.

Mode 4: Clown Stone.
Because who has more soul on the station than the Clown? Mostly good for traps.
Help Intent: Pie Cannon. The pies contain random chemicals in them, so it's ideal that you don't fire this in your face. Very small chance to blast off a clown car.
Harm Intent: Spawns The Traps!
Grab Intent: Infects the target's soul with Clownington's Degenerative Disorder, causing them to slowly become a clown who can barely fight back. There is no cure, unlike the Server Stone's effects.
Disarm Intent: Sprays Water (not lube, that'd just get annoying) all over the floor.

Mode 5: Lag Stone.
The bane of Carbonhell's existence, the Lag Stone mode mostly exists to help you get out of bad situations fast. It simply speeds you up while in use, slowly draining your stamina similar to Changeling's Strained Muscles. But you can still pack a punch.
Help Intent: Attempts to teleport you back to a set point on the station, and heals you drastically. You will be stunned for a great time after using it, and it has some startup, so it is NOT for free. Alternatively, you can use this alongside GBJ to create a safehouse, or a massive prison if used on your foes! The main use for the mode, outside of running away. You may say this is more fitting for the Bluespace Stone butidontcarefuckyou
Harm Intent: Space Ninja's weird teleport slicey thing. I don't know the name or the rules.
Grab Intent: Fires a projectile, and swaps places with the victim!
Disarm Intent: Shoots a projectile that will create a ton of dizzying effects on the target.

"Mode" 6: Powergame Stone.
Name suggested by Maroon, who is a moron. The joke being that Powergame is all in the mind.
All Intents: Lasers, because if there's anything that reminds you of powergame it's tase + lase.

### Spells
Obviously, Wiz starts off with the classic MM and EJ. He also starts off with three other spells:
Shockwave, Super Punch, Regeneration and Charge. What do they have to do with Thanos? Beats me, but he has super strength and borderline immortality so give me some creative liberty.
Each and every spell (for now) will be listed below:
**Shockwave:** Spawns a massive shockwave, knocking opponents away from the user and to the ground for a short time. A useful "get off of me" tool.
Is buffed up immensely once you have the Syndie Stone, has a possibility of setting people on fire.
**Super Punch:** Punch so hard you send a massive wave flying forward incredibly quickly, dealing heavy damage on contact.
Once you have the Syndie Stone, you will [learn how to use your biceps](https://github.com/HippieStation/HippieStation/pull/5091), which makes the already powerful shot even better, especially because your biceps are much better than those puny normal Wizards who can only launch people like three tiles away.
**Charge:** No, not like the recharge spell, this is just a rod-form-esque bull rush forward. Anyone who gets hit gets pummeled.
Once you have the Syndie Stone, it'll let you smash through walls and go even faster while pummeling harder. Once in use, you'll also be invincible to all damage in its duration.
**Regeneration:** If you had to pussy out of a fight, then this is a good way to recuperate. Requires that you stand still for an extended time, but will heal very admirably without the need of other chemicals or those pesky doctors.

### Spells - Syndie Stone
See above.

### Spells - Bluespace Stone
**Bluespace Jaunt:** Replaces Ethereal Jaunt. It moves much quicker, allowing you to cover way more space than usual, and still works when stunned.
**Bluespace Blink:** A blink spell with much more range, but leaves behind a tracer of where you went. Don't worry, you can breathe in space.
**Portal Shield:** Summons a shield in your left hand which will negate any and all damage coming your way for 15 seconds, or until you get hit 5 times. Attacks are sent wildly across bluespace.
### Spells - Server Stone
**Gondola Morph:** Upon contact with a target, turns them into a gondola! If the gondola is killed, they will pop back out and return to their regular selves.
**Reality Distortion:**
https://github.com/HippieStation/HippieStation/blob/c87165263ea55d1a0b1a82f52adbcd1f67729a2c/code/modules/spells/spell_types/spacetime_distortion.dm
**Summon GBJ:** Creates an inflatable jail exactly where you stand. This can either be used as a safe home for recovery, or as a prison for victims. Once this spell is used, it's gone forever.
inflatable house

### Spells - Honk Stone
**Rise Up:** Returns a dead body to a completely subservient cluwne! Obviously worse than a Golem, but cheaper.
**Prankster's Delusion:** Causes those around you to see others as a clumsy clown, including yourself! Now how will they know who is who?

### The Stones
Once War is openly declared, the stones will be delivered to certain crewmembers or scattered about. Crewmembers in possession of a stone will radiate a glow or aura that lets everyone know of their importance, including the Wizard if he swaps modes on his Gauntlet.
Stones _should_ attempt to roll for the person with the highest chance of getting said stone first, i.e the Clown will be rolled for the Honk Stone first before the Captain. Everyone starts off with a flat 10% chance for each stone, but can only get one stone. On top of that, every stone also has a modifier giving some jobs a better shot at it. If every single roll fails, the Stone should randomly respawn in maint similar to a Nuke Disk.
If someone bearing a stone is killed, it is immediately removed from their possession and dropped on the ground ripe for the taking. Cremating or spacing a stone will merely have it respawn on the station, again like the Nuke Disk.
Modifiers are as follows:
**Syndie Stone:**
HoS: 70% chance. Captain: 45% chance. Security Officers: 20% chance. Head of Personnel: 15% chance.
The person who has the Syndie Stone gains a martial art similar to the Default MA, and passively recovers from injuries over time.

**Bluespace Stone:**
RD: 60% chance. Scientists: 20% chance. Mime: 15% chance (because he's all about running away!)
The person who has the Bluespace Stone gains a considerably weaker Bluespace Blink, and gains the Bluespace Prophet brain "trauma."

**Server Stone:**
CE: 60% chance. Librarian: 45% chance. Engineers: 30% chance.
The person who has the server stone basically just has telekinesis and x-ray vision because I don't have any ideas and am a boring person.

**Honk Stone:**
Clown: 90% chance. Librarian: 20% chance. Mime: 0% chance.
The person who has the Honk Stone can summon banana peels out of thin air, prime for the chucking.

**Lag Stone:**
Quartermaster: 40% chance. Other Cargonians (not miners): 20% chance.
The bearer of the Lag Stone gains the same ability as the Gauntlet's Help Intent, but it only works on themselves and whoever they have grabbed. It also has longer startup.

**Powergame Stone:**
Captain: 70% chance. Janitor: 25% chance.
The bearer of the Powergame Stone gains the Recharge spell, as well as being able to communicate with ghosts via Scrying Orb-like spells.

Once all stones are handed out, the crew is given around 5-10 minutes to prepare for a Wizard attack. Cargo is also rewarded with additional points. **Because war is declared, I strongly suggest that if this ever gets merged, admins ban any sort of grief. No reason to act like a dick in decwar.**

## Once all stones have been collected by the Wiz they're meant to be practically unstoppable. And of course, they get the famed snap to kill half of the station population.

[Why]: # How does your feature improve the game anyhow? Why should it be added?
Because Wizard has no real means of declaration of war. Simply put, he only has Summon Guns and Magic, which aren't actually challenges for the Wizard. They merely make the crew murderbone each other and drive admins to drink. Any replacement for that is ideal, especially after /tg/ nerfed it into the ground.
Really, this whole idea is just a fun/funny way of seeing what could be done with Declaration of War with a Wizard. DecWar is underappreciated for how much fun or chaotic it can be, thanks to our population having been in decline for a while and Nuke Ops barely happening any more. But now that we have a population, I think it's time we could add something to Wizard for that purpose.
Besides, tis all optional. Wizards can still be massive lazy dingos and drive admins to drink with Summon Guns and Magic.

[Resources]: # Eventual assets such as icons and sounds which you have and want to suggest related to this feature.
Obviously the gauntlet sprite. I can handle the rest.

[Have fun coding this, jackass]: 
The hardest choices require the strongest wills.
thanos_smile
But really, I'm definitely biting off more than I can chew in one go. I just want Wizard to have a cool item that works as Declaration of War material.

# Be sure to ask Carbonhell to label this as a feature and not -1 it off the bat because it's big. Either ping them on our discord or PM them on our forum. Thank you for sharing your ideas! BRB re-adding shadowling.
@HippieStation/maintainers

### PR DESCRIPTION
**Pull request by ghost**
closes https://github.com/HippieStation/HippieStation/issues/9145

:cl: 
add: The Badmin Gauntlet!
add: Wizards, for all their spellpoints, can now declare war by buying the Badmin Gauntlet!
add: Their objective is to collect all 6 Infinity Stones and snap out half the station!
fix: Autoshotguns should no longer be lag guns
fix: Admins can now delete stationloving stuff
/:cl:

### CODE REVIEW

**Comment by Manly-Dwarf** [comment]
Pre-approved, in case we (you more like) have to change too much, I'll +1 it any way.

Things I'd probably advise changing would be moving the Z function to C, and letting you instantly select a mode rather than constantly toggle. This also gives you the ability to give modes another ability bound to Z (ASSUMING YOU'RE SMART AND MAKE THE INFINITY GAUNTLET AN IN-HAND LIKE A GOODBOY). Go nuts.
If you need any ideas or want to check up with stuff, message me and I'll help you out.

**Comment by Manly-Dwarf** [comment]
Already liking the progress big time, steamp0rt. Good stuff. I'll hopefully have some sprites for you soon.

File: hippiestation/code/modules/gauntlet/bluespace.dm [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+		user.remove_trait(TRAIT_PACIFISM, BLUESPACE_STONE_TRAIT)
+		user.remove_movespeed_modifier(BLUESPACE_STONE_TRAIT)
+		animate(user, alpha = 255, time = 15)
+	else
+		user.incorporeal_move = INCORPOREAL_MOVE_BASIC
+		user.add_trait(TRAIT_PUSHIMMUNE, BLUESPACE_STONE_TRAIT)
+		user.add_trait(TRAIT_IGNOREDAMAGESLOWDOWN, BLUESPACE_STONE_TRAIT)
+		user.add_trait(TRAIT_STUNIMMUNE, BLUESPACE_STONE_TRAIT)
+		user.add_trait(TRAIT_SLEEPIMMUNE, BLUESPACE_STONE_TRAIT)
+		user.add_trait(TRAIT_PACIFISM, BLUESPACE_STONE_TRAIT) // just so this doesn;t get abused to fire guns while invincible
```
Comment (Manly-Dwarf) [inline hippiestation/code/modules/gauntlet/bluespace.dm]: nice catch

File: hippiestation/code/game/objects/items/holotool/holotool.dm [hunk 2]
```diff
@@ -118,4 +118,4 @@
 	if(chosen)
 		var/new_tool = LAZYACCESS(mode_names, chosen)
 		if(new_tool)
-			switch_tool(user, new_tool)
\ No newline at end of file
+			switch_tool(user, new_tool)
```
Comment (ghost) [inline hippiestation/code/game/objects/items/holotool/holotool.dm:121]: stupid newline

**Comment by bigbluebeaker** [comment]
+1

**Comment by ghost** [comment]
Getting close to completion...

**Comment by ghost** [comment]
Should be able to finish up tonight

**Comment by ghost** [comment]
Then it'll just be a wait for sprites to be done

**Comment by ghost** [comment]
all that's needed now is QoL tweaks, the armor/species, and balancing

**Comment by ExpAntique** [comment]
If you pull this off I'll officially take you off my hit list

 +1.

**Comment by ghost** [comment]
#### Test Merge Deployed

##### Server Instance
HippieStation

##### Revision
Origin: cf7d75cdc4b9761e2902cdaabc33ed6a37d6708b
Pull Request: d0cb6f902673d7c6756d9567b3d68ee4fcfbc766
Server: 373cef429e08fdae5b25967e85000717e716e340

##### Comment
okay let's do this

**Comment by ghost** [comment]
#### Test Merge Removed

**Comment by ghost** [comment]
#### Test Merge Deployed

##### Server Instance
HippieStation

##### Revision
Origin: cf7d75cdc4b9761e2902cdaabc33ed6a37d6708b
Pull Request: cf0a6717c467d39a9c351751a0f0340fa7c34b25
Server: b6f0a3a824892e00860720523a23aa89f8c9eab8

##### Comment
why is testmerging so terrible

**Comment by ghost** [comment]
#### Test Merge Removed

**Comment by sagaty12** [comment]
Hey man I told you I had some ideas so here they are

-  Avengers ERT. They get automatically called if Thanus gets most stones way too early in the round. 

- There's already some stuff in the game code that can create most Avengers like the Wizard Mjolnir + Recall spell ~and a beer keg~ for Thor, a powerful mech for Iron Man ~with an exosuit badmin gauntlet like in the movie? hell yeah that would be cool~ and Hulk is pretty obvious.

- When you nuke the station/summon nar'sie theres a "cinematic" ending so why not have one for this too? Basically my idea is to force the ghosts to observe the victims of the snap one by one (NOT like the ghost magnet, more like when theres a meteor random event and ghosts get a prompt to orbit the meteors) [with that creepy SFX](https://www.youtube.com/watch?v=anBOuaxwDZI) for moar immursion


ye I know my ideas suck but why not

**Comment by Pringleschipman** [comment]
> Hey man I told you I had some ideas so here they are
> 
> * Avengers ERT. They get automatically called if Thanus gets most stones way too early in the round.
> * There's already some stuff in the game code that can create most Avengers like the Wizard Mjolnir + Recall spell ~and a beer keg~ for Thor, a powerful mech for Iron Man ~with an exosuit badmin gauntlet like in the movie? hell yeah that would be cool~ and Hulk is pretty obvious.
> * When you nuke the station/summon nar'sie theres a cinematic ending so why not have one for this too? Basically my idea is to force everyone to observe the victims of the snap one by one [with that creepy SFX](https://www.youtube.com/watch?v=anBOuaxwDZI)
> 
> ye I know my ideas suck but why not

add in black panther and youve got me. need the added opportunities for racism.

**Comment by sagaty12** [comment]
> add in black panther and youve got me. need the added opportunities for racism.

Don't forget black widow cause I respect women :100:

**Comment by mikolaj007** [comment]
So:
-hulk has hulk
-thor has mjolnir + instant summons
-black ~~panther~~ cat is a black tajaran with gloves of north star
-Black widow is a literal spider
-captain nanotransen  should get a badass shield and instant summons
-the griffin gets a crossbow with some rods
-Steel man gets some cool armor

**Comment by sagaty12** [comment]
> -black ~panther~ cat is a black tajaran with gloves of north star

valid :smile_cat: 
> -Black widow is a literal spider

what about spiderman though
:thinking:

**Comment by mikolaj007** [comment]
>
> what about spiderman though
> 🤔

a second spider obviously

**Comment by sagaty12** [comment]
the chain of command has a special attack that looks like something spiderman would do
trips and pulls the victim, does no damage so its not op

**Comment by ghost** [comment]
#### Test Merge Deployed

##### Server Instance
HippieStation

##### Revision
Origin: 06f2efe93f8da6dad7478dcfb918aa39f6a0dcaa
Pull Request: 2aef39282b593421a73dbd7dde0ae44969281d3a
Server: 0fad314f41f6ebac5ea74539092beed6a139976f

##### Comment
another test round

**Comment by NautilusViper** [comment]
+1 fucking merge it forever already

**Comment by Bondle** [comment]
-1

**Comment by Majick1** [comment]
+1

**Comment by ghost** [comment]
#### Test Merge Removed

**Comment by ExpAntique** [comment]
Suggestion: After the snap, let whoever’s playing Thanos open a portal to the farm

**Comment by ghost** [comment]
@ExpAntique good idea

**Comment by DaydreamPuppeh** [comment]
I am inevitable

**Comment by sagaty12** [comment]
this should be a gamemode or something
its perfect

**Comment by cb583** [comment]
+1
Destiny still arrives

**Comment by ghost** [comment]
#### Test Merge Deployed

##### Server Instance
HippieStation

##### Revision
Origin: fb7861a0503c3a61c00519f76b4df9824c24a612
Pull Request: ef40d911f83eb4ecc6aba1605260eae548e42c88
Server: 2907375bc90421e2043cc8375e53d05ab145956c

##### Comment
Round Three

**Comment by ghost** [comment]
#### Test Merge Removed

File: hippiestation/code/modules/gauntlet/ghost.dm [hunk 3]
```diff
@@ -0,0 +1,355 @@
...
+		BK.name = "cluwne's cursed knife"
+		H.put_in_hands(BK, TRUE)
+		H.visible_message("<span class='danger'>[H] struggles back up, now a cluwne!</span>")
+		to_chat(H, "<span class='userdanger'>You are risen from the dead as a cluwne. [caller] is your master. Follow their orders at all costs.</span>")
+		Finished()
+	else
+		revert_cast()
+
+/obj/effect/proc_holder/spell/targeted/conjure_item/spellpacket/sandmans_dust
+	name = "Ghost Stone: Sandman's Dust"
```
Comment (GuyonBroadway) [inline hippiestation/code/modules/gauntlet/ghost.dm:301]:
```suggestion
	name = "Ghost Stone: Sandman's Dust"
	invocation = "POCKET SAND"
	invocation_type = "shout"
```

**Comment by yobob591** [comment]
once it works +1, needs a bit more testing

**Comment by ghost** [comment]
#### Test Merge Deployed

##### Server Instance
HippieStation

##### Revision
Origin: 3b3644d8904786dcc3da7e6d1151336ec88ca793
Pull Request: 5902817a5a66ee98ee12d0df8ed0fc1cfed793b4
Server: edf3fa80402fd38fb6689aa74277aad25da1022b

##### Comment
THE FINAL TESTMERGE??? MAYBE???

**Comment by ghost** [comment]
#### Test Merge Updated

##### Server Instance
HippieStation

##### Revision
Origin: 0328fb3fad55c1d8c3f307f77e908df0a80a580d
Pull Request: 5902817a5a66ee98ee12d0df8ed0fc1cfed793b4
Server: 8ed8b4e3efb33463fc314670b4fba27c1a6353e7

##### Comment
okay for real for real last test

**Comment by ghost** [comment]
#### Test Merge Updated

##### Server Instance
HippieStation

##### Revision
Origin: 0328fb3fad55c1d8c3f307f77e908df0a80a580d
Pull Request: 4f362c5a3b1649973ca9cfe75d9c039012926b47
Server: 380ef45e55de6ce21b356589740c41794d59bfac

##### Comment
Update.

**Comment by yobob591** [comment]
big +1 def ready

**Comment by Majick1** [comment]
+1 let's do it

**Comment by ghost** [comment]
#### Test Merge Updated

##### Server Instance
HippieStation

##### Revision
Origin: 3d2041e78faed35e25a6ed58660d79269879d2eb
Pull Request: 5bab52456d0822a04283ac881b55bce9d5156d9f
Server: ce119c5818b6cd54848549e06fe1577a9271d77d

##### Comment
Update.

**Comment by yobob591** [comment]
I'd like to request something, a version of the gauntlet that doesnt spawn stones or declare war when put on for admin shitposting like sending thanos deathsquad

**Comment by yobob591** [comment]
Also, the infinity gauntlet can quickly destroy rwalls but requires you to bash doors repeatedly to break them

**Comment by ghost** [comment]
#### Test Merge Updated

##### Server Instance
HippieStation

##### Revision
Origin: 3d2041e78faed35e25a6ed58660d79269879d2eb
Pull Request: 6dc6cc5c62aa535f743f4b36e9c9e85efe49adfb
Server: f835a98e2da0470f9f0399c3f083fa6b5348f455

##### Comment
Update.

**Comment by ghost** [comment]
#### Test Merge Updated

##### Server Instance
HippieStation

##### Revision
Origin: 639d8342c103ff2d3197d882ba1a2f558cfd5edc
Pull Request: c2cf864ebaad88cf3a530ea4b1c38a229a6a5f16
Server: 2d4004947c6b98cec6342fb128073dd57e026f86

##### Comment
Update.

**Comment by AffectedArc07** [comment]
Me likey

I dont even play here but I like this

**Comment by ghost** [comment]
don't you dare steal this for [insert server you're currently with], affected

**Comment by k-k-karma** [comment]
> Me likey
> 
> I dont even play here but I like this

fag

**Comment by AffectedArc07** [comment]
@steamp0rt you think I have enough energy for that? Im halfway through my finals

@k-k-karma no u

**Comment by Couls** [comment]
too late, an artifact this powerful is meant to be shared

**Comment by ghost** [comment]
#### Test Merge Updated

##### Server Instance
HippieStation

##### Revision
Origin: 8ad798873bf2ab73025ced8f9d2c1b347777407f
Pull Request: 57c343d04123678a892eb7f8210aeaec3d7d6a56
Server: f15ffa120cb482cd80e7baa245907dbefb362220

##### Comment
Update.

**Comment by Manly-Dwarf** [comment]
GET OUT OF YOUR CODE YOU YOGSTATION CODE THIEVES

File: hippiestation/code/modules/gauntlet/_stone.dm [hunk 4]
```diff
@@ -0,0 +1,271 @@
+/obj/item/infinity_stone
```
Comment (GuyonBroadway) [inline hippiestation/code/modules/gauntlet/_stone.dm]:
```suggestion
//Originally coded for hippie by Steamp0rt, shared under agpl.
/obj/item/infinity_stone
```

Comment (GuyonBroadway) [reply hippiestation/code/modules/gauntlet/_stone.dm]: Because leagally speaking under agpl people cannot remove inline citations

**Comment by ExpAntique** [comment]
Petition to make hippiestation closed source for a month to shoo away yogstation code thieves

**Comment by sagaty12** [comment]
I've isolated the snap SFX directly from the movie for you baby

[snap.zip](https://github.com/HippieStation/HippieStation/files/3209768/snap.zip)

**Comment by GuyonBroadway** [comment]
See now that there are accreditation comments in the code if anyone else wants to use this they legally have to keep them in as part of agpl.

**Comment by ATH1909** [comment]
Is there a check to see if the wizard has used their magic mirror to change their race BEFORE putting on the infinity gauntlet?

**Comment by ghost** [comment]
@ATH1909 using the gauntlet force changes your race to thanos anyways

**Comment by ghost** [comment]
#### Test Merge Updated

##### Server Instance
HippieStation

##### Revision
Origin: ccbb0797d45c02daba153bf3f739bc6692ed8482
Pull Request: 889a9da39efe35467af8df42dbcef68d7809289a
Server: 5baa9fb2622c54665648dc3228fa9876fdf86054

##### Comment
Update.

**Comment by ghost** [comment]
#### Test Merge Updated

##### Server Instance
HippieStation

##### Revision
Origin: ddefe65b5112604c6cb77552c64f834822ebab68
Pull Request: 889a9da39efe35467af8df42dbcef68d7809289a
Server: d2dd9f12f17ed24a28967e180f9777084154d85f

##### Comment
Update.

**Comment by ghost** [comment]
#### Test Merge Updated

##### Server Instance
HippieStation

##### Revision
Origin: 108a220378df96784ad3942bc39f6e03092b2dc7
Pull Request: 321e63a234e36633afd11cbc398868b6513dd3a9
Server: 9183cb939885116804ce2012e418f1cda6d682bb

##### Comment
Update.

**Comment by ghost** [comment]
#### Test Merge Updated

##### Server Instance
HippieStation

##### Revision
Origin: 108a220378df96784ad3942bc39f6e03092b2dc7
Pull Request: 0f0afbd22cbcf25556dd4d57ad5dc2d40ce008c8
Server: bbcc1d2312a2a1d5b6d7f4506fd105c3d722f46c

##### Comment
Update.

**Comment by ghost** [comment]
#### Test Merge Updated

##### Server Instance
HippieStation

##### Revision
Origin: 310e178d24316b7bafd8fb77b42565e8dc9cbaf9
Pull Request: bd351861935ce9980e01fe96bdf93ffc0042e3ea
Server: 02b05aea4ffc32cfa5d48a33661638f28bd07f8c

##### Comment
Update.

**Comment by ghost** [comment]
#### Test Merge Updated

##### Server Instance
HippieStation

##### Revision
Origin: 310e178d24316b7bafd8fb77b42565e8dc9cbaf9
Pull Request: 51a955ab3eee27c2c37c08cde10445059e0cc9d7
Server: 5d86e320745dfc523914d064e23a024751406e5d

##### Comment
Update.

**Comment by ghost** [comment]
#### Test Merge Updated

##### Server Instance
HippieStation

##### Revision
Origin: 67f4df41d842068f6939850ca543accbebc8e70a
Pull Request: fbebe351208db58d8c959337391358f0e8158d0c
Server: da33c0edc22b8a30f695a874fd2fbcf9bf49e0da

##### Comment
Update.

**Comment by AffectedArc07** [comment]
image

**Comment by k-k-karma** [comment]
image

**Comment by ghost** [comment]
#### Test Merge Updated

##### Server Instance
HippieStation

##### Revision
Origin: 0bfcae94d9525e97fb95971516e5e6d5021b179e
Pull Request: 3127536a3bb8148ee6fda6173db60679edba2d18
Server: 714e784899af510034cd480241d09a0d5d6c0e96

##### Comment
Update.

**Comment by ghost** [comment]
#### Test Merge Updated

##### Server Instance
HippieStation

##### Revision
Origin: fab9e0a430a814745f75fb6d8c8bd0c7cd97e499
Pull Request: e7ff742f191c0362e60760b8f9c3227a9b7f9c0e
Server: 55988d3df4950419d290a98544a5338f7daa801e

##### Comment
Update.

**Comment by ghost** [comment]
#### Test Merge Updated

##### Server Instance
HippieStation

##### Revision
Origin: bdeebde36b0d3765ef1c29e99833ca5c1df91cba
Pull Request: e7ff742f191c0362e60760b8f9c3227a9b7f9c0e
Server: 2b2e4fa80eb2e64094b10643c3f6ccd28c664eff

##### Comment
Update.

**Comment by ghost** [comment]
#### Test Merge Updated

##### Server Instance
HippieStation

##### Revision
Origin: 37ec11dc828cdc1ee3239e38aa84bcaec28760bc
Pull Request: 2ec1cde23a9cc3c8d092165e6898e7ef8842b686
Server: bb872e4741299c73900cc8710d72f5dbbe749fc0

##### Comment
Update.

**Comment by ghost** [comment]
#### Test Merge Updated

##### Server Instance
HippieStation

##### Revision
Origin: 21fecc8817b538b8bf27c1d9da8fb6a93d5a6477
Pull Request: 2ec1cde23a9cc3c8d092165e6898e7ef8842b686
Server: 6687fed1ab416bc53d60fa2ac5cc264892366687

##### Comment
Update.

**Comment by ExpAntique** [comment]
Please add:

 “PROTIP: To defeat Thanos, shoot at him until he dies”

To the lobby tip pool

**Comment by Manly-Dwarf** [comment]
Waiting on the farm now.

**Comment by ghost** [comment]
#### Test Merge Updated

##### Server Instance
HippieStation

##### Revision
Origin: e6f9bc0c1632d0be33755a613cefb5e2aaff4732
Pull Request: a33fbfe78a452bc4454a5cda312d76347d0e9c4a
Server: cb7d7109fa95fbe2805c02e0e7dc9ae69497c761

##### Comment
Update.

**Comment by ghost** [comment]
#### Test Merge Updated

##### Server Instance
HippieStation

##### Revision
Origin: dc05d74f3dd7344ebebe4697fa69108b969ace58
Pull Request: 653171853948f2e54e593d0d5e33b12aaf35b363
Server: 56dffd5b15c3d0072fa3f7298fcca339487db1d5

##### Comment
Update.

**Comment by ghost** [comment]
#### Test Merge Updated

##### Server Instance
HippieStation

##### Revision
Origin: 1a2f3cb65fa92f3586a20c50efa8ed20e257fb7b
Pull Request: 7a56ba86391c81d507c7535ed458668f0ca1c269
Server: 44b8fe368f82921e929f0690f17ba9738271e5db

##### Comment
Update.

**Comment by ghost** [comment]
#### Test Merge Updated

##### Server Instance
HippieStation

##### Revision
Origin: 1a2f3cb65fa92f3586a20c50efa8ed20e257fb7b
Pull Request: e351e832360dad0e6200e6005ac1126efdd7deed
Server: dd0298ee74c908edd73ef9fdcb12c86a9eb23c3b

##### Comment
steamp0rt forgot his scroll I guess

**Comment by sagaty13** [comment]
avengers ERT is so broken lmao wheres hulk green skin

**Comment by sagaty13** [comment]
badmin gauntlet needs a golden painting for **immersion**
also killing thanos should drop the gauntlet

**Comment by ghost** [comment]
@sagaty13 hulk not being green is fixed.

and you can gib thanos in order to get the gauntlet.

**Comment by ghost** [comment]
#### Test Merge Updated

##### Server Instance
HippieStation

##### Revision
Origin: 91a4d5a438db997d704c9295343e7bf40f2e7e76
Pull Request: 1ec6103ed164167818e5d78811bf801ea0bf74c5
Server: f76779e756f0af1d6bf287b1bcb75f58f6236a3b

##### Comment
adds farm

**Comment by ghost** [comment]
#### Test Merge Updated

##### Server Instance
HippieStation

##### Revision
Origin: cbba97d1e020f6e4bd96e3ea814a08c18327746a
Pull Request: e95e32ab06a7f23a9c3e6dce23e62e9e83226646
Server: 56a21609227b77d5e0dfa4544cc220f189707f6a

##### Comment
buffs the car, some other changes

**Comment by ghost** [comment]
#### Test Merge Updated

##### Server Instance
HippieStation

##### Revision
Origin: 260977d467c52f5dea6e352a3a1e80be2f33c29f
Pull Request: f197a424f609e1f7440258ab42c101ea48cbbc51
Server: 1f18fc6380271376dfc8b87dad10baaa830da60f

##### Comment
fixes a bug related to amari forgetting that not everyone is supposed to be an avenger

**Comment by ghost** [comment]
#### Test Merge Updated

##### Server Instance
HippieStation

##### Revision
Origin: 5eaf83fc7ee6da6e983ca4aa6dcaf6860ed7c091
Pull Request: 37370421437c16070b2ba6cec5961ae62890538d
Server: 5ab6746c26daf358ed3fffbc36c509778800f019

##### Comment
fixes a bug related to amari forgetting that not everyone is supposed to be an avenger

**Comment by ghost** [comment]
#### Test Merge Updated

##### Server Instance
HippieStation

##### Revision
Origin: 2dd783999e68cb02be34d39cf007191bc2aa3020
Pull Request: 38ff2f22c20b2a83840f930b1d33da33a61854a1
Server: 10690eb9c9a485be1f048b6cabf73762685ce46e

##### Comment
fixes a bug related to amari forgetting that not everyone is supposed to be an avenger

**Comment by ghost** [comment]
lol travis failed due to some unrelated error

**Comment by ghost** [comment]
#### Test Merge Updated

##### Server Instance
HippieStation

##### Revision
Origin: 0ee43a0bbbc714feea0c72f2210acbdfe54f403c
Pull Request: fdaa513902a5d432df9375f0c35306e5ab277a52
Server: 15dbd32e88a02f20c79abda5c2e911d4f4092bcf

##### Comment
fixes a bug related to amari forgetting that not everyone is supposed to be an avenger

**Comment by ghost** [comment]
#### Test Merge Updated

##### Server Instance
HippieStation

##### Revision
Origin: 4c868e55d4c73698cd125dabdfb495a259abe0e9
Pull Request: 2755682ce943c9265c44129ce416db4e05ce4bb1
Server: 3213677f08408477e90a4672a2b50537a7780062

##### Comment
Quick testmerge update, can we merge this yet?

**Comment by ghost** [comment]
#### Test Merge Updated

##### Server Instance
HippieStation

##### Revision
Origin: e863436548af86bf7930b2c9ba3226bc7a5a8a71
Pull Request: 6f464c2cc5d35a5727207ad51278df177b403cfe
Server: 7296bf2a7db6d8580e5e4cac709d9d20238f66dc

##### Comment
kills conflicts

**Review by yoyobatty — APPROVED** [review]
Merge time!

**Comment by ghost** [comment]
#### Test Merge Removed

**Comment by sagaty13** [comment]
dude you gotta seriously buff the revengers ERT before merging this lmao they suck ass

1. Captain Nanotrasen shield doesn't do any damage only slips but Thanus can't be slipped
2. Hulk is even worse, dude is so weak that hes always the first one to get killed
3. I sent you so many ideas and sprites and you didn't implement them yet so theres a lot of placeholders

**Pull request merged**